src/coreclr/vm/ceeload.h

```diff
@@ -677,7 +621,6 @@ struct ModuleCtorInfo
 
 #ifdef FEATURE_PREJIT
```

Comment:
this could be deleted too?

Reply:
I'm planning on doing a pass of removing everything under `FEATURE_PREJIT` in a separate change. (This change happens to do some of it because those two defines were rather intertwined in some places)